src/results/db.rs

```diff
@@ -51,6 +51,7 @@ impl<'a> DatabaseDB<'a> {
                 &base64::decode(&result.log).with_context(|_| "invalid base64 log provided")?,
                 encoding_type,
             )?;
+            self.mark_crate_as_completed(ex, &result.krate)?;
```

Comment:
This is called two times per crate, one for each toolchain.

Reply:
This still needs to be addressed.

Reply:
Well, it's still called two times per crate but this does not cause any problem. To be sure if it's the first toolchain run or the second I must check the database and this is what the function is now doing.

Reply:
Hmm, ok.